grader/ajax_key.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
Hm.. This and `secret_key.py` should probably be in .gitignore :thinking: . Neither of them should be in the git.